client/ayon_max/plugins/load/load_max_scene.py

```diff
@@ -161,12 +160,23 @@ def update(self, container, context):
                                      current_max_object_names):
             max_obj.name = f"{namespace}:{obj_name}"
             max_objects.append(max_obj)
-            max_transform = f"{max_obj}.transform"
+            max_transform = f"{max_obj.name}.transform"
             if max_transform in transform_data.keys():
```

Comment:
The keys is redundant for a dict lookup?
```suggestion
            if max_transform in transform_data:
```